test/e2e/specs/editor/various/block-bindings.spec.js

```diff
@@ -1,2451 +0,0 @@
-							url_custom_field: imageCustomFieldSrc,
-						},
-					},
-				} );
-				await page.reload();
-			} );
-			test( 'should show the value of the custom field when url is bound', async ( {
-				editor,
-			} ) => {
-				await editor.insertBlock( {
-					name: 'core/image',
-					attributes: {
-						anchor: 'image-url-binding',
-						url: imagePlaceholderSrc,
-						alt: 'default alt value',
-						title: 'default title value',
-						metadata: {
-							bindings: {
-								url: {
-									source: 'core/post-meta',
-									args: { key: 'url_custom_field' },
-								},
-							},
-						},
-					},
-				} );
-				const imageBlockImg = editor.canvas
-					.getByRole( 'document', {
-						name: 'Block: Image',
-					} )
-					.locator( 'img' );
-				await expect( imageBlockImg ).toHaveAttribute(
-					'src',
-					imageCustomFieldSrc
-				);
-
-				// Check the frontend uses the value of the custom field.
-				const previewPage = await editor.openPreviewPage();
-				const imageDom = previewPage.locator(
-					'#image-url-binding img'
-				);
-				await expect( imageDom ).toHaveAttribute(
-					'src',
-					imageCustomFieldSrc
-				);
-				await expect( imageDom ).toHaveAttribute(
-					'alt',
-					'default alt value'
-				);
-				await expect( imageDom ).toHaveAttribute(
-					'title',
-					'default title value'
-				);
-			} );
-
-			test( 'should show value of the custom field in the alt textarea when alt is bound', async ( {
-				editor,
-				page,
-			} ) => {
-				await editor.insertBlock( {
-					name: 'core/image',
-					attributes: {
-						anchor: 'image-alt-binding',
-						url: imagePlaceholderSrc,
-						alt: 'default alt value',
-						title: 'default title value',
-						metadata: {
-							bindings: {
-								alt: {
-									source: 'core/post-meta',
-									args: { key: 'text_custom_field' },
-								},
-							},
-						},
-					},
-				} );
-				const imageBlockImg = editor.canvas
-					.getByRole( 'document', {
-						name: 'Block: Image',
-					} )
-					.locator( 'img' );
-				await imageBlockImg.click();
-
-				// Image src is the placeholder.
-				await expect( imageBlockImg ).toHaveAttribute(
-					'src',
-					imagePlaceholderSrc
-				);
-
-				// Alt textarea should have the custom field value.
-				const altValue = await page
-					.getByRole( 'tabpanel', { name: 'Settings' } )
-					.getByLabel( 'Alternative text' )
-					.inputValue();
-				expect( altValue ).toBe( 'Value of the text custom field' );
-
-				// Check the frontend uses the value of the custom field.
-				const previewPage = await editor.openPreviewPage();
-				const imageDom = previewPage.locator(
-					'#image-alt-binding img'
-				);
-				await expect( imageDom ).toHaveAttribute(
-					'src',
-					imagePlaceholderSrc
-				);
-				await expect( imageDom ).toHaveAttribute(
-					'alt',
-					'Value of the text custom field'
-				);
-				await expect( imageDom ).toHaveAttribute(
-					'title',
-					'default title value'
-				);
-			} );
```

Comment:
Unify all the image attributes in this test for custom sources: https://github.com/WordPress/gutenberg/pull/65526/files#diff-70af8def41b67ce618e94f6b4f68044bb4540816a1a8084a1ba1eac9e0c26595R141-R215

It is missing the title.